apps/prairielearn/src/ee/pages/instructorAiGenerateJobDetails/instructorAiGenerateJobDetails.html.ts

```diff
@@ -0,0 +1,79 @@
+import { html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+
+import { HeadContents } from '../../../components/HeadContents.html.js';
+import { Job } from '../../../lib/db-types.js';
+
+export function AiGenerateJobDetailsPage({
+  resLocals,
+  job,
+}: {
+  resLocals: Record<string, any>;
+  job: Job;
+}) {
+  return html` <!doctype html>
+    <html lang="en">
+      <head>
+        ${HeadContents({ resLocals })}
+      </head>
+      <body hx-ext="loading-states">
+        ${renderEjs(import.meta.url, "<%- include('../../../pages/partials/navbar'); %>", {
+          navPage: 'course_admin',
+          ...resLocals,
+        })}
+        <main id="content" class="container-fluid">
+          <div class="card mb-4">
+            <div class="card-header bg-primary text-white">Generation Job Results</div>
+            <div class="card-body">
+              <div class="mr-auto">
+                <span class="card-title"> Prompt </span>
+              </div>
```

Comment:
Is the `mr-auto` wrapper really necessary?

Consider using a semantic heading (e.g. `<h2>`) instead of just a span. You can apply one of Bootstrap's [heading classes](https://getbootstrap.com/docs/5.3/content/typography/#headings) to make the size more visually appealing.

Reply:
removed in 3c8bafd1470728a8c79b488213de1dfae0aed79e